amundsen_application/static/js/ducks/dashboard/reducer.ts

```diff
@@ -0,0 +1,92 @@
+
+
+/* Actions */
+import { GetDashboard, GetDashboardRequest } from 'ducks/dashboard/types';
+import { Dashboard } from 'interfaces/Dashboard';
+import { User } from 'interfaces/User';
+import { TableSummary } from 'interfaces/TableMetadata';
+import { Tag } from 'interfaces/Tags';
+import { TableMetadataReducerState } from 'ducks/tableMetadata/reducer';
+
+export function getDashboard(uri: string, searchIndex?: string, source?: string): GetDashboardRequest {
```

Comment:
nit: We could start introducing objects here since that's what we want to move towards:
```
export function getDashboard({ uri: string, searchIndex?: string, source?: string }): 
```